cypress/e2e/spec.cy.js

```diff
@@ -21,37 +21,65 @@ describe('template spec', () => {
     cy.url().should('eq', 'http://localhost:3000/result')
   })
   it("should show ResultPage with header, salary, salary after taxes, annual expenses, tax on expenses, zipcode, final salary, and take user home when link is clicked", () => {
+    // cy.intercept({
+    //   method: 'GET', // or 'POST' or any other HTTP method
+    //   url: 'https://u-s-a-sales-taxes-per-zip-code.p.rapidapi.com/12345',
+    //   // statusCode: 200,
+    //   headers: {
+    //     'X-RapidAPI-Key': '8e3c03a57bmshcf9c926514fc323p15ca95jsna29d71f8a66a',
+    //     'X-RapidAPI-Host': 'u-s-a-sales-taxes-per-zip-code.p.rapidapi.com'
+    //   }
+    //   // fixture: 'example.json'
+    // }).as('fetchRequest')
+    // cy.intercept('GET', 'https://u-s-a-sales-taxes-per-zip-code.p.rapidapi.com/12345', {
+		// 	statusCode: 200,
+		// 	fixture: 'example.json'
+		// });
     cy.intercept({
-      method: 'GET', // or 'POST' or any other HTTP method
-      url: 'https://u-s-a-sales-taxes-per-zip-code.p.rapidapi.com/12345',
-      // statusCode: 200,
+      method: 'GET',
+      url: 'https://u-s-a-sales-taxes-per-zip-code.p.rapidapi.com/*', // Use wildcard for dynamic parts like zipcode
       headers: {
         'X-RapidAPI-Key': '8e3c03a57bmshcf9c926514fc323p15ca95jsna29d71f8a66a',
         'X-RapidAPI-Host': 'u-s-a-sales-taxes-per-zip-code.p.rapidapi.com'
       }
-      // fixture: 'example.json'
-    }).as('fetchRequest')
-    // cy.intercept('GET', 'https://u-s-a-sales-taxes-per-zip-code.p.rapidapi.com/12345', {
-		// 	statusCode: 200,
-		// 	fixture: 'example.json'
-		// });
-    cy.visit('http://localhost:3000/result')
+    }, (req) => {
+      // Customize the response or stub the response data as needed
+      req.reply({
+        statusCode: 200,
+        fixture: 'example.json'
```

Comment:
Had to make sure that the req.reply(the data we're snubbing) is the same format as a normal snubbing